test_loop.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
```

Comment:
This also needs a license header. However, looking at the script, it should not be in this PR (it's in the root) ?

Reply:
oups...